doc/guide/magnetism/magnetism.rst

```diff
@@ -3,104 +3,153 @@
-    M_{\perp z'} &= M_{0z} \\
-    M_{0q_x} &= (M_{0x}\cos\phi - M_{0y}\sin\phi)\cos\phi \\
-    M_{0q_y} &= (M_{0y}\sin\phi - M_{0x}\cos\phi)\sin\phi
+    \mathbf{M}_{\perp,\perp P } = \mathbf{M}_{\perp } - (\mathbf{P}\cdot \mathbf{M}_{\perp }) \mathbf{P}
+
 
-Here, $M_{0x}$, $M_{0x}$, $M_{0z}$ are the x, y and z components
-of the magnetization vector given in the laboratory xyz frame given by
+and a third vector perpendicular to both $\mathbf{P}$ and $ \mathbf{M}_{\perp,\perp P } $ :
 
 .. math::
-    M_{0x} &= M_0\cos\theta_M\cos\phi_M \\
-    M_{0y} &= M_0\sin\theta_M \\
-    M_{0z} &= -M_0\cos\theta_M\sin\phi_M
+    \mathbf{O} = \mathbf{M}_{\perp} \times \mathbf{P} - \mathbf{M}_{\perp,\perp P }
+
+
+
+For symmetric, collinear spin structures ($\mathbf{M}_{\perp}^{\ast}=\mathbf{M}_{\perp}^{\ast}$).
+
+
+
+Every magnetic scattering cross section can be constructed from an incoherent mixture
+of the 4 spin-resolved spin states depending on the efficiency parameters before
+($u_i$) and after ($u_f$) the sample. For a half-polarised experiment(SANSPOL with $u_f=0.5$) or
+full (longitudinal) polarisation analysis, the accessible spin states are measured
+independently and a simultaneous analysis of the measured states is performed,
+tying all the model parameters together except $u_i$ and $u_f$, which are set based
+on the (known) polarisation efficiencies of the instrument.
+
+.. note::
+    The values of the 'up_frac_i' ($u_i$) and 'up_frac_f' ($u_f$) must be in the range 0 to 1.
+    The parameters 'up_frac_i' and 'up_frac_f' can be easily associated to
+    polarisation efficiencies 'e_in/out' (of the instrument). Efficiency values range from 0.5
+    (unpolarised beam)  to 1 (perfect optics) or 0 (perfect optics, but other spin state). For 'up_frac_i/f'  <0.5
+    a cross section is constructed with the spin reversed/flipped with respect
+    to the initial supermirror polariser. The actual polarisation efficiency
+    in this case is however  'e_in/out' = 1-'up_frac_i/f'.
+
+
 
-and the magnetization angles $\theta_M$ and $\phi_M$ are defined in
-the figure above.
 
 The user input parameters are:
 
 ===========   ================================================================
- sld_M0       $D_M M_0$
+ sld_M0       $b_H M_0$
  sld_mtheta   $\theta_M$
  sld_mphi     $\phi_M$
- up_frac_i    $u_i$ = (spin up)/(spin up + spin down) *before* the sample
- up_frac_f    $u_f$ = (spin up)/(spin up + spin down) *after* the sample
- up_angle     $\theta_\mathrm{up}$
+ up_frac_i    $u_i$ polarisation efficiency *before* the sample
+ up_frac_f    $u_f$ = polarisation efficiency *after* the sample
+ p_theta      $\theta_P$
+ p_phi        $\phi_P$
 ===========   ================================================================
 
-.. note::
-    The values of the 'up_frac_i' and 'up_frac_f' must be in the range 0 to 1.
+
+
+
+
+References
+----------
+
+    .. [#] R. M. Moon and T. Riste and W. C. Koehler, *Phys. Rev.*, 181 (1969) 920.
```

Comment:
This is causing a warning when building the docs, which causes the build to fail:
```
    2021-03-17T20:53:13.1935777Z /home/runner/work/sasmodels/sasmodels/doc/guide/magnetism/magnetism.rst:148: WARNING: Footnote [#] is not referenced.
```
The simple solution is to just drop `    .. [#] ` from the beginning of the line. The other solution is to reference it somewhere in the text.

Reply:
How is the rst compilation different than the compilation of the documentation in the python models?  there references do not use a directive (`..[#] author_name`) but simply a numbered bullet (`#. author_name...`) why does that not work here?

Reply:
The warning is that the footnote is not referenced. Using a simple numbered list (`#. author_name...`) should work fine.

Elsewhere in the document (e.g., refs.rst) we do not use numbered lists, but simple paragraphs one after the other.